softwareComponents/simplesimClientLib/include/simplesim_client.hpp

```diff
@@ -21,13 +21,26 @@
 
 #include "atoms/guarded.hpp"
 #include "configuration/rofibot.hpp"
+#include "configuration/serialization.hpp"
+#include "legacy/configuration/IO.h"
```

Comment:
I prefer to extract Matrix formatter into a separate header/library. Feel free to use fmt::fmt for that.

Reply:
I prefer to have this implemented before merging (otherwise we will totally forget about it).